content/keywords/pt_PT.yaml

```diff
@@ -1,12 +1,12 @@
 add: adicionar
 and: e
 ask: perguntar
-at: em
+at: no
```

Comment:
The build is failing because the default behavior of the yaml parser is to convert some words to boolean values. So `no` becomes False which is why we get `TypeError: can only concatenate str (not "bool") to str`. I guess the solution is to wrap the value in quotes (i.e. `'no'`) but I am not sure how to do that without breaking weblate flow further. @jpelay do you know how to change it?